examples/ScatteringExample/src/G4patcheDPWACoulombScatteringModel.cc

```diff
@@ -0,0 +1,191 @@
+// * LICENSE and available at  http://cern.ch/geant4/license .  These *
+// * include a list of copyright holders.                             *
+// *                                                                  *
+// * Neither the authors of this software system, nor their employing *
+// * institutes,nor the agencies providing financial support for this *
+// * work  make  any representation or  warranty, express or implied, *
+// * regarding  this  software system or assume any liability for its *
+// * use.  Please see the license in the file  LICENSE  and URL above *
+// * for the full disclaimer and the limitation of liability.         *
+// *                                                                  *
+// * This  code  implementation is the result of  the  scientific and *
+// * technical work of the GEANT4 collaboration.                      *
+// * By using,  copying,  modifying or  distributing the software (or *
+// * any work based  on the software)  you  agree  to acknowledge its *
+// * use  in  resulting  scientific  publications,  and indicate your *
+// * acceptance of all terms of the Geant4 Software license.          *
+// ********************************************************************
+//
+//
+// -------------------------------------------------------------------
+//
+// GEANT4 Class header file
+//
+//
+// File name:     G4patcheDPWACoulombScatteringModel
+//
+// Author:        Mihaly Novak
+//
+// Creation date: 02.07.2020
+//
+// Modifications: 16.04.2021 YR: lines 89/90 removed
+//
+// -------------------------------------------------------------------
+
+#include "G4patcheDPWACoulombScatteringModel.hh"
+
+#include "G4eDPWAElasticDCS.hh"
+#include "G4ParticleChangeForGamma.hh"
+#include "G4ParticleDefinition.hh"
+#include "G4DataVector.hh"
+
+#include "G4ProductionCutsTable.hh"
+#include "G4Material.hh"
+#include "G4Element.hh"
+#include "G4ElementVector.hh"
+
+#include "G4Electron.hh"
+
+#include "G4PhysicalConstants.hh"
+#include "G4SystemOfUnits.hh"
+#include "Randomize.hh"
+#include "G4ThreeVector.hh"
+
+
+G4patcheDPWACoulombScatteringModel::G4patcheDPWACoulombScatteringModel(G4bool ismixed, G4bool isscpcor, G4double mumin)
+: G4VEmModel("eDPWACoulombScattering"),
+  fIsMixedModel(ismixed),
+  fIsScpCorrection(isscpcor),
+  fMuMin(mumin),
+  fTheDCS(nullptr),
+  fParticleChange(nullptr)
+{
+  SetLowEnergyLimit (  0.0*CLHEP::eV);  // ekin = 10 eV   is used if (E< 10  eV)
+  SetHighEnergyLimit(100.0*CLHEP::MeV); // ekin = 100 MeV is used if (E>100 MeV)
+}
+
+
+G4patcheDPWACoulombScatteringModel::~G4patcheDPWACoulombScatteringModel()
+{
+  if (IsMaster()) {
+    delete fTheDCS;
+  }
+}
+
+
+void G4patcheDPWACoulombScatteringModel::Initialise(const G4ParticleDefinition* pdef,
+                                               const G4DataVector& prodcuts)
+{
+  if(!fParticleChange) {
+    fParticleChange = GetParticleChangeForGamma();
+  }
+  // fMuMin        = 0.5*(1.0-std::cos(PolarAngleLimit()));
```

Comment:
Is it worth raising this with the Geant4 devs? Is there an active bug report? (I looked, but couldn't find one)

Reply:
patched version removed. Still think it is wrong to overwrite input parameter but at least it doesn't matter.